shared-libs/couch-request/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@medic/couch-request",
+  "version": "1.0.0",
+  "description": "Provides a declarative global options interface for request-promise-native",
```

```suggestion
  "description": "Provides a declarative global options interface for http requests",
```

Let's not tie ourselves to rpn!